mxcube3/ui/components/SampleQueue/TaskItem.js

```diff
@@ -196,20 +196,21 @@ export default class TaskItem extends Component {
             <form>
               <div className="form-group row">
                 <label className="col-sm-12">File path:</label>
-                <div className="col-sm-12" style={{ display: 'flex' }}>
+                <div className="col-sm-12" style={{ display: 'flex' }} >
```

Comment:
A few comments, at least on this code - I did not have time to review the whole PR yet:
- as we rely on react-bootstrap we should replace all old-style `form`+`input` etc to proper react-bootstrap equivalents, as it was done when we did the upgrade of libraries
- in React we should avoid as much as possible accessing the DOM tree directly with `document.getElementById` for example - there are references (callback refs) for doing this kind of thing
- we should avoid too much CSS styling (what is `display: flex` doing here ? Maybe it can be replaced by some options in react-bootstrap components ?)